web/app.js

```diff
@@ -1330,6 +1330,78 @@ var PDFViewerApplication = {
       });
     }
   },
+
+  unbindEvents: function pdfViewBindEvents() {
+    var eventBus = this.eventBus;
+
+    eventBus.off('resize', webViewerResize);
+    eventBus.off('hashchange', webViewerHashchange);
+    eventBus.off('beforeprint', this.beforePrint.bind(this));
```

Comment:
.bind() returns different function every time it is called, but eventBus.on and .off require the same method to be used.